pkgdb/src/buildenv/command.cc

```diff
@@ -40,6 +62,11 @@ BuildEnvCommand::BuildEnvCommand() : parser( "buildenv" )
     .metavar( "SYSTEM" )
     .nargs( 1 )
     .action( [&]( const std::string & str ) { this->system = str; } );
+
+  this->parser.add_argument( "--container", "-c" )
+    .help( "build a container" )
+    .nargs( 0 )
+    .action( [&]( const auto & ) { this->buildContainer = true; } );
 }
```

Comment:
nit: might be good to change the help text on `--out-link` to be `path to link resulting environment or container builder script`